### PR TITLE
Add Leetcode 001,104

### PR DESCRIPTION
**Pull request by zmzhouXJTU**

### CODE REVIEW

**Comment by jovan-wong** [comment]
@zmzhouXJTU  Thanks your work on this, and I left 2 small suggestions:
1. I think you should rebase the master branch firstly.
2. split this PR into 2 small PRs: we should request an PR for one solution in my opinion.

File: 001._two_sum.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            target(int): the value that we assign
+        
+        Returns:
+            the indices of the two numbers from the array
+        """    
+        for i in range(len(nums)):
+            for j in range(i+1, len(nums)):
+                if nums[i] + nums[j] == target:
+                    return [i, j]
+
```
Comment (stormsunshine) [inline 001._two_sum.md]: What if there is no solution? Although in the description it is assumed that there is exactly one solution, it is better to add a return statement at the end of this function that returns an empty list.

File: 001._two_sum.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            nums(List[int]): the name of array of integers
+            target(int): the value that we assign
+        
+        Returns:
+            the indices of the two numbers from the array
+        """    
+        for i in range(len(nums)):
+            for j in range(i+1, len(nums)):
+                if nums[i] + nums[j] == target:
+                    return [i, j]
```
Comment (stormsunshine) [inline 001._two_sum.md]: What if there is no solution? Although in the description it is assumed that there is exactly one solution, it is better to add a return statement at the end of this function that returns an empty list.

Comment (zmzhouXJTU) [reply 001._two_sum.md]: OK，Thanks for your review. I will correct it later

**Comment by zmzhouXJTU** [comment]
@jovan-wong  OK. Thanks for your advice

**Pull request closed without merging**